modules/social_features/social_profile/modules/social_profile_preview/social_profile_preview.module

```diff
@@ -83,20 +83,6 @@ function social_profile_preview_preprocess_field__field_profile_image(array &$va
   }
 }
 
-/**
- * Implements hook_preprocess_HOOK().
- */
-function social_profile_preview_preprocess_flag(array &$variables): void {
-  if (\Drupal::routeMatch()->getRouteName() === 'social_profile_preview.canonical') {
-    $variables['attributes']['class'][] = 'btn-primary';
-    $classes = &$variables['attributes']['class'];
-
-    if (($delta = array_search('btn-default', $classes)) !== FALSE) {
-      unset($classes[$delta]);
-    }
-  }
-}
-
```

Comment:
Sorry, but when I said to remove it, I just forgot that this module hasn't a dependency on the follow user module. Please, restore this code.